radio/src/gui/480x272/layout.h

```diff
@@ -113,11 +113,12 @@ class BaseLayoutFactory: public LayoutFactory
     const ZoneOption * options;
 };
 
-#define MAX_REGISTERED_LAYOUTS 10
-
-extern unsigned int countRegisteredLayouts;
-extern const LayoutFactory * registeredLayouts[MAX_REGISTERED_LAYOUTS];
 Layout * loadLayout(const char * name, Layout::PersistentData * persistentData);
 void loadCustomScreens();
 
+#define MAX_REGISTERED_LAYOUTS          10
+extern unsigned int countRegisteredLayouts;
+#define registeredLayouts      getRegisteredLayouts()
```

Comment:
is it possible you remove this #define, it's a little bit confusing, I would prefer we use your new function everywhere as a singleton